src/web/jcc/client/components/CommandControl/CommandControl.tsx

```diff
@@ -3122,6 +3123,9 @@ export default class CommandControl extends React.Component {
                 >
                     <Icon path={mdiPlay} title="Run Mission" />
                 </Button>
+                <Button id="downloadAll" className={`button-jcc`} onClick={this.processDownloadAllBots.bind(this)}>
```

Comment:
It's because we need to bind the method to the value of "this" at that point.  onClick just expects a function with 1 argument (an Event object) or 0 arguments, and wouldn't know what "this" was without the bind.